gh-runner/Dockerfile

```diff
@@ -32,6 +33,10 @@ RUN curl -L -O https://github.com/actions/runner/releases/download/v${GH_RUNNER_
 COPY --from=phoenixrtos/devel /opt/phoenix/utils /opt/phoenix/utils
 ENV PATH="/opt/phoenix/utils:${PATH}"
 
+# get JLink tools
+COPY --from=phoenixrtos/devel /opt/SEGGER /opt/SEGGER
```

Comment:
what happens when we're building gh-runner for x86? (this folder currently doesn't exist in `phoenixrtos/devel:amd64` image)

Reply:
There is the following error:
![Screenshot from 2022-09-20 13-49-44](https://user-images.githubusercontent.com/77304431/191250146-49e4316d-ff74-48ed-935b-eef5cf07ade3.png)

I can propose the following workaround before adding jlink tools for x86:
`# get JLink tools
COPY --from=dloew/phoenixrtos:armv7 /opt/SEGGER /opt/SEGGER
ENV PATH="/opt/SEGGER/JLink:${PATH}"`

It seems to work fine.

Reply:
> > > please also update `CHANGELOG.md` as a part of the PR (until new docker image release it can be placed under eg. `NEXT VERSION`)
> > 
> > 
> > Changes applied, waiting only for [phoenix-rtos/phoenix-rtos-project#502](https://github.com/phoenix-rtos/phoenix-rtos-project/pull/502) merge to specify the proper `phoenix-rtos-project` commit.
> 
> No need to do so, the commit hash will be written when next docker images would be built (by the builder)

So can I leave `* phoenix-rtos-project commit: `xxxxxxx``?

Reply:
- no, don't copy `arm` binaries into `amd64` image - they won't work! You have 3 choices - install JLink also for other architectures, create empty `/opt/SEGGER` dir in `devel:amd64` image or only optionally copy the files while building `gh-runner` image
- changelog - just omit this line - just list Your changes. I'll add commit hash when I'll be preparing the release.

Reply:
Yeah, I knew that it won't work, but wanted to find out some quick solution. However after your comment I've realized that it's quite simple to add it for amd64 at once. So it's added and I've tested it by starting jlink gdb server for nrf9160 (`JLinkGDBServer -device nrf9160 -if swd -port 2331`). It works, but requires `--privileged` argument when starting the image :)